apps/bedrock-content-generator/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file is conflicting with the repo level prettier config (https://github.com/contentful/apps/blob/master/.prettierrc) and as a result we're seeing a build failure.

Let's remove this file entirely from this PR.

If your local hooks are set up this should trigger an automatic formatting of all the files post commit to fix all the mismatches. To do this manually you can, from the project root, run `npx prettier -w apps/bedrock-content-generator` and it will manually apply the prettier fixes for you.

Reply:
Thank you!